src/Microsoft.Data/PrimitiveColumn.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Data
+{
+    /// <summary>
+    /// A column to hold primitive values such as int, float etc. Other value types are not really supported
```

Comment:
What does `Other value types are not really supported` mean?

Reply:
I wanted to say that only primitive types are really supported. A user defined struct for example will throw if some binary op is called. I removed the last sentence and changed it to now say "A column to hold primitive types such as int, float etc"